subjects/admin_views.py

```diff
@@ -29,7 +29,7 @@ class ModifySubject(forms.Form):
 def subject_add(request, sub_id=0, sub_name=''):
     # create the traversal object
     traversal = TraversalUtils()
-    if not request.user.is_authenticated():
+    if not request.user.is_authenticated:
```

Comment:
While we are changing these from using the method, I suggest we change them to be more explicit. As part of continuous education security training requirements, I took Secure Coding in Python which mentions to avoid truthy/falsy comparisons when checking authentication with `is_authenticated`. I think we should use `if user.is_authenticated is not True` instead.